cdap-standalone/src/main/java/io/cdap/cdap/StandaloneMain.java

```diff
@@ -147,6 +148,7 @@ public class StandaloneMain {
   private final OperationalStatsService operationalStatsService;
   private final TwillRunnerService remoteExecutionTwillRunnerService;
   private final MetadataSubscriberService metadataSubscriberService;
+  private final AppDeletionSubscriberService appDeletionSubscriberService;
```

Comment:
putting it only here means it will only run in Standalone. AppFabricServer is probably a better place.